fission/src/ui/panels/configuring/initial-config/InputSchemeSelection.tsx

```diff
@@ -33,6 +33,9 @@ function InputSchemeSelection({ brainIndex, onSelect, onEdit, onCreateNew }: Inp
 
                 {/** Creates list items with buttons */}
                 {InputSchemeManager.availableInputSchemes.map(scheme => {
+                    if (scheme.usesTouchControls && !matchMedia("(hover: none)").matches) {
+                        return <></>
```

Comment:
```suggestion
                        return null
```

I would replace with null instead of returning an empty fragment node.